doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -156,14 +156,13 @@ TODO Write a set of cookbook documents and link to them.
      priv = BN_bin2bn(priv_data, sizeof(priv_data), NULL);
 
      param_bld = OSSL_PARAM_BLD_new();
-     if (param_bld != NULL) {
-         OSSL_PARAM_BLD_push_utf8_string(param_bld, "curve-name",
-                                         "prime256v1", 0);
-         OSSL_PARAM_BLD_push_BN(param_bld, "priv", priv);
-         OSSL_PARAM_BLD_push_octet_string(param_bld, "pub",
-                                          pub_data, sizeof(pub_data));
+     if (priv != NULL && param_bld != NULL
+         && OSSL_PARAM_BLD_push_utf8_string(param_bld, "curve-name",
```

Comment:
Ok so you missed one very important change..
Which is the 'curve-name' got changed to 'group'. Which is why it is arguably better to use the #defines instead.

Reply:
Also - my actual data for the key data works - as opposed to your crappy values that are not valid :)

Reply:
Ah, thanks.  I had actually found that by testing the code myself, but apparently forgot to update that string.